content/ja/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -0,0 +1,104 @@
+---
+title: サイドカーコンテナ
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.29" state="beta" >}}
+
+サイドカーコンテナは、メインのアプリケーションコンテナと同じ{{< glossary_tooltip text="Pod" term_id="pod" >}}内で実行されるセカンダリーコンテナです。
+これらのコンテナは、主要なアプリケーションコードを直接変更することなく、
+ロギング、モニタリング、セキュリティ、データの同期などの追加サービスや機能を提供することにより、
+アプリケーションコンテナの機能を強化または拡張するために使用されます。
+
+<!-- body -->
+
+## サイドカーコンテナの有効化
+
+デフォルトでKubernetes 1.29に有効化された[フィーチャーゲート](/docs/reference/command-line-tools-reference/feature-gates/)という機能は、
+`SidecarContainers`と名付けられ、Podの`initContainers`フィールドに記載されているコンテナの`restartPolicy`を指定することができます。
+これらの再起動可能な_サイドカー_コンテナは、
```

Comment:
```suggestion
これらの再起動可能な _サイドカー_ コンテナは、
```

Reply:
Thank you!!

This is because rendering fails if half-spaces are not included.